pkg/nodegraph/Library/NodegraphApp.js

```diff
@@ -9,7 +9,7 @@
 import {App} from '../../Library/App/App.js';
 import {LocalStoragePersistor} from '../../Library/App/storage.js';
 import {RecipeService} from '../../Library/App/RecipeService.js';
-import {StoreUpdateService} from '../../Library/NodeGraph/StoreUpdateService.js';
+import {StoreService} from '../../Library/NodeGraph/StoreService.js';
```

Comment:
As above, all of StoreService and RecipeService is handled in ArcsWorker.

Reply:
i reverted the NodegraphApp file, nodegraph/ needs more work, will do a follow up pr

Reply:
somehow the revert doesn't show properly.. anyway... follow up pr